third-upgrades/content/config/creatures/dungeon/lvl4Upg2.json

```diff
@@ -105,7 +105,7 @@
 				"bonus" :
 				{
 					"type" : "SPELL_AFTER_ATTACK",
-					"subtype" : 70, // stoneGaze
+					"subtype" : "spell.stoneGaze",
 					"addInfo" : 2000 // FIXME: replace with range?
```

Comment:
Very old format should be replaced with "addInfo" : [1,1]

Reply:
done